patient/README.md

```diff
@@ -0,0 +1,97 @@
+# Patient Mapping
+
+## JSON FHIR > XML HL7
+
+Patient is mapped to an entry.
+
+| Mapped to (JSON FHIR Immunization field)        | Mapped from (XML HL7 / other source)                                                              |
+|-------------------------------------------------|---------------------------------------------------------------------------------------------------|
+| resourceType.id                                 | set value = Generated GUID                                                                        |
+| resourceType.meta.profile                       | fixed value = `"https://fhir.nhs.uk/STU3/StructureDefinition/GPConnect-StructuredRecord-Bundle-1"`|
+| resourceType.type                               | fixed value = `collection`                                                                        |
+| resourceType.entry[resource.id]                 | fixed value = `{{nhs-number}}`                                                                    |
+| resourceType.entry[resource.meta.id]            | fixed value = `1521806400000`                                                                     |
+| resourceType.entry[resource.meta.profile]       | fixed value = `https://fhir.nhs.uk/STU3/StructureDefinition/CareConnect-GPC-Patient-1`            |
+| resourceType.entry[resource.indentifier.system] | fixed value = `https://fhir.nhs.uk/Id/nhs-number`                                                 |
+| resourceType.entry[resource.indentifier.value]  | fixed value = `5538824210`                                                                        |
+
+
+## Example JSON
+
+<details>
+```
+{
+    "resourceType": "Bundle",
+    "id": "373e9ffc-827e-479d-80fa-ba5814d084b8",
+    "meta": {
+        "profile": [
+            "https://fhir.nhs.uk/STU3/StructureDefinition/GPConnect-StructuredRecord-Bundle-1"
+        ]
+    },
+    "type": "collection",
+    "entry": [
+        {
+            "resource": {
+                "resourceType": "Patient",
+                "id": "82c11ce2-7d9f-49a7-930b-0a195e1e9775",
+                "meta": {
+                    "versionId": "1521806400000",
+                    "profile": [
+                        "https://fhir.nhs.uk/STU3/StructureDefinition/CareConnect-GPC-Patient-1"
+                    ]
+                },
+                "identifier": [
+                    {
+                        "system": "https://fhir.nhs.uk/Id/nhs-number",
+                        "value": "5538824210"
+                    }
+                ],
+            }
+        }

```

Comment:
I think these are the relevant lines, I think we can exclude the bundle part above as that's generic?